cves/kernel/CVE-2019-12378.yml

```diff
@@ -186,8 +187,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: There was no mention of a violation.
```

Comment:
I'd also check if the code itself reflected this.

Reply:
updated